templates/psi4.yaml

```diff
@@ -27,8 +27,6 @@ spec:
         default: 'False'
       - name: nthreads
         default: 1
-      - name: n-active
```

Comment:
Could you please add `save-rdms` (default: False) to the template parameter list and to the function call (`save_rdms = {{inputs.parameters.save-rdms}}` )?

Reply:
Addressed here: c515960

Reply:
Awesome, thanks!